app/models/edition.rb

```diff
@@ -37,6 +37,7 @@ class Edition < ApplicationRecord
   has_many :authors, through: :edition_authors, source: :user
   has_many :classification_featurings, inverse_of: :edition
   has_many :link_check_reports, as: :link_reportable, class_name: LinkCheckerApiReport
+  has_many :links_reports, as: :link_reportable
```

Comment:
I assume the plan here is to remove this in a further PR? Could we add a comment just to explain this is temporary.